modules/protocol-tests/test/src/smithy4s/api/validation/ErrorMessageTraitValidatorSpec.scala

```diff
@@ -159,7 +159,6 @@ object ErrorMessageTraitValidatorSpec extends FunSuite {
         .assemble()
 
     val expected = List(
-      noErrorTrait(union),
```

Comment:
**question:** not sure what recent changes made this test fail because of this. Maybe these 2 Smithy validations have been separated and are not reported in parallel: traits validations and custom validations

Reply:
I think that makes sense 👍